### ISSUE TITLE
Failed to compile the source.

### ISSUE THREAD

**Issue by hongyi-zhao** [direnv/direnv#548]
Hi,

I cloned the source code of direnv and then try to make it, but failed with the following error:
```

$ make 
cd "/home/werner/.env/direnv/direnv.git/.gopath/src/github.com/direnv/direnv" && go build  -o direnv
go: cannot find main module, but found Gopkg.lock in /home/werner/.env/direnv/direnv.git/.gopath/src/github.com/direnv/direnv
	to create a module there, run:
	go mod init
make: *** [GNUmakefile:70: direnv] Error 1
```

**Comment by hongyi-zhao** [comment]
I am not familiar with golang.  After do some googling for this issue and solved the issue with the following method:

$ go mod init github.com/direnv/direnv
$ make

Regards

**Comment by zimbatm** [comment]
thanks for the report. This is fixed by #552
